api/indexer/server.go

```diff
@@ -44,6 +45,62 @@ func (f *apiFactory) New(vm api.VM) (api.Handler, error) {
 	}, nil
 }
 
+type GetBlockRequest struct {
+	BlockID ids.ID `json:"blockID"`
+}
+
+type GetBlockByHeightRequest struct {
+	Height uint64 `json:"height"`
+}
+
+type GetBlockResponse struct {
+	Block      *chain.ExecutedBlock `json:"block"`
+	BlockBytes codec.Bytes          `json:"blockBytes"`
+}
```

Comment:
Alternative here is to include the requested encoding in the request

Reply:
It might create a mess. There aren't that many methods. It's fine to duplicate unless we plan to increase the number of methods.